pandas/core/groupby.py

```diff
@@ -187,11 +187,19 @@ class Grouper(object):
 
     Examples
     --------
-    >>> df.groupby(Grouper(key='A')) : syntactic sugar for df.groupby('A')
-    >>> df.groupby(Grouper(key='date',freq='60s')) : specify a resample on the column 'date'
-    >>> df.groupby(Grouper(level='date',freq='60s',axis=1)) :
-        specify a resample on the level 'date' on the columns axis with a frequency of 60s
+    
+    Syntactic sugar for `df.groupby('A')`
```

Comment:
you better use double backticks instead of single ones around `df.groupby()` (to format it as code, with single it is formatted in italics)